src/silo/storage/column/indexed_string_column.cpp

```diff
@@ -50,4 +81,12 @@ IndexedStringColumnPartition& IndexedStringColumn::createPartition() {
    return partitions.emplace_back(*lookup);
 }
 
+void IndexedStringColumn::generateLineageIndex(const common::LineageTreeAndIDMap& lineage_tree) {
+   *lookup = lineage_tree.lineage_id_lookup_map.copy();
+   for (auto& partition : partitions) {
+      std::reference_wrapper(partition.lookup) = *lookup;
+      partition.lineage_index = LineageIndex(lineage_tree.lineage_tree);
```

Comment:
Could you please explain why we need this reference wrapper here?

Reply:
If you think it's beneficial also as a comment in the code. But mostly for my understanding of those two lines here (a PR comment would be enough ;) ).